src/Nox.Generator/Domain/EntityMeta.template.cs

```diff
@@ -14,11 +14,30 @@ namespace {{codeGeneratorState.DomainNameSpace}};
 /// </summary>
 public partial class {{className}}
 {
-{{- for entry in propertiesWithSource }}
+{{- for td in typeDef }}
```

Comment:
we can improve readability, changing the name for example
```suggestion
{{- for entityMetadata in entitiesMetadata}}
```

Reply:
Desired changes were applied

Reply:
lets solve this on a second PR